R/insulation_score.R

```diff
@@ -67,6 +67,7 @@ insulation_score <- function(explist, window = 30,
   outer_bins <- CJ(V1 = bins, V2 = (ext_window - 1):(window))
   outer_bins <- outer_bins[, grp := V1 + V2 - window]
   inner_bins <- outer_bins[, list(V1, V2 = V2 - window, grp)]
```

Comment:
Since a recent update in data.table, the `inner_bins` variable no longer has V1 and V2 as keys. I presume that is because we have mutated V2.